src/objects/aff/zcl_abapgit_object_common_aff.clas.abap

```diff
@@ -218,12 +219,23 @@ CLASS ZCL_ABAPGIT_OBJECT_COMMON_AFF IMPLEMENTATION.
             messages = <ls_messages>.
 
         LOOP AT <ls_messages> ASSIGNING <ls_message>.
+          ASSIGN COMPONENT 'MESSAGE' OF STRUCTURE <ls_message> TO <ls_msg>.
           ASSIGN COMPONENT 'TEXT' OF STRUCTURE <ls_message> TO <ls_text>.
           ASSIGN COMPONENT 'TYPE' OF STRUCTURE <ls_message> TO <ls_type>.
           ii_log->add(
               iv_msg  = <ls_text>
               iv_type = <ls_type>
               is_item = ms_item ).
+
+          CHECK <ls_msg>-msgty = 'E'.
```

Comment:
https://github.com/SAP/styleguides/blob/main/clean-abap/CleanABAP.md#avoid-check-in-other-positions

prefer using IF

Reply:
agree

Reply:
Will replace the other CHECK statements as well